demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -79,6 +110,7 @@ export function ProductsGrid() {
             flex: 1,
             sortBy: ["title", "price", "type", "category", "inStock"],
             visible: theme.breakpoints.down("md"),
+            valueGetter: ({ value, row }) => row.title, // needed for export because renderCell does not return string
```

Comment:
"overview" column should not be exported at all, we already have a title (and others) columns

Reply:
there will probably be cases where we've got combining columns, without separate additional columns for all contained information. should adding an excel export require adding those separate columns, probably hidden and never used in the web-version, to allow exporting this data or should there be an option to export multiple columns based on a single column-config?

Reply:
yes, of course we will then add those as separate columns. Imho there is no need for making them unusable in the web-version, we just hide them by default.

> or should there be an option to export multiple columns based on a single column-config?

No. At least not for now.